js/src/server/privileges.js

```diff
@@ -461,6 +461,20 @@ AJAX.registerOnload('server/privileges.js', function () {
         addOrUpdateSubmenu();
     }
 
+    var tableSelectAll = function (e) {
+        e.preventDefault();
+        var method = $(this)[0].getAttribute('data-method');
```

Comment:
```suggestion
        var method = $(this).first().data('method');
```
That should work too
And avoid index errors